package/mediacenter-addon-osmc/files/DEBIAN/postinst

```diff
@@ -41,13 +41,16 @@ then
 	if ! is_rbp; then rm -rf /usr/share/kodi/addons/script.module.osmcsetting.pi* >/dev/null 2>&1; fi
 	if is_atv; then rm -rf /usr/share/kodi/addons/script.module.osmcsetting.updates >/dev/null 2>&1; fi
 	if ischroot; then OWNER="1000:1000"; else OWNER="osmc:osmc"; fi
-	chown -R $OWNER /usr/share/kodi/addons/script.module.elementtree
+	rm -rf /usr/share/kodi/addons/script.module.elementtree >/dev/null 2>&1
 	chown -R $OWNER /usr/share/kodi/addons/script.module.xmltodict
 	chown -R $OWNER /usr/share/kodi/addons/script.module.osmcsetting.*
 	chown -R $OWNER /usr/share/kodi/addons/service.osmc.settings
+	chmod +x /usr/bin/grab-logs
+	chmod +x /usr/bin/readgui
+	chmod +x /usr/bin/remote-cli
 	if is_rbp && [ -n "$2" ] && dpkg --compare-versions "$2" lt "3.0.678"
 	then
-		/usr/bin/env python /usr/share/kodi/addons/script.module.osmcsetting.pi/resources/lib/OSMC_REparser.py
+		/usr/bin/env python /usr/share/kodi/addons/script.module.osmcsetting.pi/resources/lib/osmcpi/osmc_reparser.py
```

Comment:
/usr/bin/env python should be /usr/bin/python3 just in case the user runs update-alternatives and changes the default python version.